### PR TITLE
Completed javascript-data-model. | Vera Yeh

### PR DESCRIPTION
**Pull request by VeraYeh**
image

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+var orderHistory = [
+  {
+    orderNumber: '114-3941689-8772232',
+    orderDate: 'August 4, 2020',
+    itemName: 'JavaScript for impatient programmers',
```
Comment (BrettAlbright) [inline javascript-data-model/order-history.js]: so the thing you want to add to this data model is a property `items` which will contain an array of objects in which each object contains an individual items data: 
```javascript
var orderHis = [
  {
    orderedDate: '2020-08-04',
    totalPrice: 34,
    shipTo: 'JS Masher',
    orderNum: '114-3941689-8772232',
    deliveredDate: '2020-08-08',
    orderedDate: '2-15-21',
    items:[{
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "electronics",
      title: "Blu Ray Player"
    },
    {
      type: "book",
      title: "How to forget"
    },
    {
      type: "book",
      title: "how to remember"
    }
  ]
  }
]
```

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Great initial submission! Small update suggested!

File: javascript-data-model/order-history.js (hunk 1 above)
Comment (VeraYeh) [reply javascript-data-model/order-history.js]: Thanks for the review and comments, codes updated.
image
image

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**